Integration tests fail intermittently

Currently all integration tests pass when ran locally, but when these are ran in a kind cluster, they fail intermittently. Given there are occations where all tests pass (such as yesterday in #1026), and others where they fail it may be due to potentially timeouts being too slow (given that integration tests actually take longer - much longer - when ran in the cluster [approx 90 minutes])

Example #1 of intermittent failures (from `jx get build logs "seldonio/seldon-core/pr-1019 #13 integration"`):

```
=================================== FAILURES ===================================
_____________________ TestRollingHttp.test_rolling_update1 _____________________

self = <test_rolling_updates.TestRollingHttp object at 0x7fcd381b3b00>

    def test_rolling_update1(self):
        run("kubectl delete sdep --all", shell=True)
        wait_for_shutdown("mymodel-mymodel-e2eb561")
        run("kubectl apply -f ../resources/graph1.json", shell=True, check=True)
        wait_for_rollout("mymodel-mymodel-e2eb561")
        print("Initial request")
        r = initial_rest_request("mymodel", "seldon")
        assert r.status_code == 200
        assert r.json()["data"]["tensor"]["values"] == [1.0, 2.0, 3.0, 4.0]
        run("kubectl apply -f ../resources/graph2.json", shell=True, check=True)
        i = 0
        for i in range(100):
            r = rest_request_ambassador("mymodel", "seldon", API_AMBASSADOR)
>           assert r.status_code == 200
E           assert 500 == 200
E             -500
E             +200

test_rolling_updates.py:35: AssertionError
_____________________ TestRollingHttp.test_rolling_update2 _____________________

self = <test_rolling_updates.TestRollingHttp object at 0x7fcd383c4ac8>

    def test_rolling_update2(self):
        run("kubectl delete sdep --all", shell=True)
        wait_for_shutdown("mymodel-mymodel-e2eb561")
        run("kubectl apply -f ../resources/graph1.json", shell=True, check=True)
        wait_for_rollout("mymodel-mymodel-e2eb561")
        print("Initial request")
        r = initial_rest_request("mymodel", "seldon")
        assert r.status_code == 200
        assert r.json()["data"]["tensor"]["values"] == [1.0, 2.0, 3.0, 4.0]
        run("kubectl apply -f ../resources/graph3.json", shell=True, check=True)
        i = 0
        for i in range(100):
            r = rest_request_ambassador("mymodel", "seldon", API_AMBASSADOR)
>           assert r.status_code == 200
E           assert 503 == 200
E             -503
E             +200

test_rolling_updates.py:69: AssertionError
_____________________ TestPythonS2iK8s.test_combiner_rest ______________________

self = <test_s2i_python.TestPythonS2iK8s object at 0x7fcd382a5630>
s2i_python_version = '0.13-SNAPSHOT'

    def test_combiner_rest(self, s2i_python_version):
        tester = S2IK8S()
>       tester.test_combiner_rest(s2i_python_version)

test_s2i_python.py:126:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
test_s2i_python.py:214: in test_combiner_rest
    check=True,
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

input = None, timeout = None, check = True
popenargs = ('kubectl apply -f ../resources/s2i_python_combiner.json',)
kwargs = {'shell': True}, process = <subprocess.Popen object at 0x7fcd382a5b38>
stdout = None, stderr = None, retcode = 1

    def run(*popenargs, input=None, timeout=None, check=False, **kwargs):
        """Run command with arguments and return a CompletedProcess instance.

        The returned instance will have attributes args, returncode, stdout and
        stderr. By default, stdout and stderr are not captured, and those attributes
        will be None. Pass stdout=PIPE and/or stderr=PIPE in order to capture them.

        If check is True and the exit code was non-zero, it raises a
        CalledProcessError. The CalledProcessError object will have the return code
        in the returncode attribute, and output & stderr attributes if those streams
        were captured.

        If timeout is given, and the process takes too long, a TimeoutExpired
        exception will be raised.

        There is an optional argument "input", allowing you to
        pass a string to the subprocess's stdin.  If you use this argument
        you may not also use the Popen constructor's "stdin" argument, as
        it will be used internally.

        The other arguments are the same as for the Popen constructor.

        If universal_newlines=True is passed, the "input" argument must be a
        string and stdout/stderr in the returned object will be strings rather than
        bytes.
        """
        if input is not None:
            if 'stdin' in kwargs:
                raise ValueError('stdin and input arguments may not both be used.')
            kwargs['stdin'] = PIPE

        with Popen(*popenargs, **kwargs) as process:
            try:
                stdout, stderr = process.communicate(input, timeout=timeout)
            except TimeoutExpired:
                process.kill()
                stdout, stderr = process.communicate()
                raise TimeoutExpired(process.args, timeout, output=stdout,
                                     stderr=stderr)
            except:
                process.kill()
                process.wait()
                raise
            retcode = process.poll()
            if check and retcode:
                raise CalledProcessError(retcode, process.args,
>                                        output=stdout, stderr=stderr)
E               subprocess.CalledProcessError: Command 'kubectl apply -f ../resources/s2i_python_combiner.json' returned non-zero exit status 1.
```

Example number # 2 from `jx get build logs "seldonio/seldon-core/pr-1019 #12 integration"`:

=================================== FAILURES ===================================
_____________________ TestRollingHttp.test_rolling_update4 _____________________

self = <test_rolling_updates.TestRollingHttp object at 0x7fe088b95be0>

    def test_rolling_update4(self):
        run("kubectl delete sdep --all", shell=True)
        wait_for_shutdown("mymodel-mymodel-e2eb561")
        run("kubectl apply -f ../resources/graph1.json", shell=True, check=True)
        wait_for_rollout("mymodel-mymodel-e2eb561")
        print("Initial request")
        r = initial_rest_request("mymodel", "seldon")
        assert r.status_code == 200
        assert r.json()["data"]["tensor"]["values"] == [1.0, 2.0, 3.0, 4.0]
        run("kubectl apply -f ../resources/graph5.json", shell=True, check=True)
        i = 0
        for i in range(50):
            r = rest_request_ambassador("mymodel", "seldon", API_AMBASSADOR)
>           assert r.status_code == 200
E           assert 500 == 200
E             -500
E             +200

test_rolling_updates.py:133: AssertionError
================== 1 failed, 30 passed in 2489.32s (0:41:29) ===================
```

--- a/testing/scripts/seldon_utils.py
+++ b/testing/scripts/seldon_utils.py
@@ -43,7 +43,7 @@ def initial_rest_request(model, namespace):
                 print("Sleeping 10 sec and trying again")
                 time.sleep(10)
                 r = rest_request(model, namespace)
-        return r
+    return r
 
 
 def create_random_data(data_size, rows=1):

